docker-multinode/master.sh

```diff
@@ -30,13 +30,13 @@ kube::multinode::detect_lsb
 
 kube::multinode::turndown
 
-kube::multinode::bootstrap_daemon
+kube::multinode::restart_docker
```

Comment:
Why should docker be restarted here?
The only case where I could see this command is between the cni config and the master/worker startup...